src/main/g8/.gitignore

```diff
@@ -0,0 +1,12 @@
+target/
+
+.settings
+.project
+.classpath
+
+.idea
+*.iml
+
+.metals
+.bloop
+
```

Comment:
The generate project gets a .gitignore with target and some other typical editor files/dirs excluded.